### DISCUSSION TITLE
[alarm] alarm-settings.yml configured alarm sending notification not effective

### DISCUSSION THREAD

**Discussion by andy7891-pixel** [apache/skywalking#11369] [General]
skywalking version and deploy by: https://github.com/apache/skywalking-helm/blob/master/chart/skywalking/README.md

my skywalking version and config:
oap:
  name: oap
  dynamicConfigEnabled: false
  image:
    repository: skywalking.docker.scarf.sh/apache/skywalking-oap-server
    tag: 9.3.0

my alarm-settings.yml config:
alarm-settings.yml: |+
rules:
  service_resp_time_rule:
    metrics-name: service_resp_time
    op: ">"
    threshold: 10
    period: 1
    count: 3
    silence-period: 5
    message: Response time of service {name} is more than 10ms in 3 minutes of last 1 minutes. test test
    hooks:
      - "feishu.default"
hooks:
  feishu:
    default:
      is-default: true
      text-template: |-
        {
          "msg_type": "text",
          "content": {
            "text": "Apache SkyWalking Alarm: \n %s."
          }
        }
      webhooks:
      - url: https://open.feishu.com/open-apis/bot/v2/hook/9e25d9ef-2ce5-4b40-9c36-3000b1480d36
        #secret: dummysecret

question:
i can see the alarm at skywalking-ui, but can't received notification by feishu. what problem?
![image](https://github.com/apache/skywalking/assets/136289470/119692ba-7e83-4f8c-97d2-d7c0d6bca53e)

**Comment by wu-sheng** [comment]
Didn't you replace secret? How is this possible to work?

**Reply by andy7891-pixel** [reply]
i don't config secret with feishu, i use ip whitelist replace secret.
```
it's work below;
curl -X POST -H'Content-Type: application/json'  https://open.feishu.com/open-apis/bot/v2/hook/9e25d9ef-2ce5-4b40-9c36-3000b1480d36 -d '{"msg_type": "text", "content": {"text": "example msg"}}'
```

**Reply by wu-sheng** [reply]
Then you need to debug why.
